src/Tribe/Main.php

```diff
@@ -1594,8 +1594,12 @@ public function saved_organizers_dropdown( $current = null, $name = 'organizer[O
 		 */
 		public function admin_body_class( $classes ) {
 			global $current_screen;
-			if ( isset( $current_screen->post_type ) &&
-				 ( $current_screen->post_type == self::POSTTYPE || $current_screen->id == 'settings_page_tribe-settings' )
+			if (
+				isset( $current_screen->post_type )
+				&& (
+					$current_screen->post_type === self::POSTTYPE
+					|| 'settings_page_tribe-settings' === $current_screen->id
+				)
```

Comment:
@jbrinley had a good idea about moving this "complex" conditional to a function in the [closed PR](https://github.com/moderntribe/the-events-calendar/pull/72#discussion_r31033115).  Here's a version of the suggestion:

```php
if ( $this->is_events_admin_screen( 'settings_page_tribe-settings' ) ) {
```

Thoughts?